src/app/button/button.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, EventEmitter, OnInit, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent implements OnInit {
+  @Input('class')
```

Comment:
I forget say you, don't make aliases, change on names o inputs for example 

@Input() class = 'primary';